src/main/webapp/components/api/api.js

```diff
@@ -9,4 +9,5 @@ angular.module('managementConsole.api', [
   'ispn.directives.cache.cacheconfiguration',
   'ispn.directives.cache.configurationsection',
   'ispn.directives.cache.jdbcstore',
+  'LocalStorageModule'
```

Comment:
Maybe we should use sessionStorage instead: https://www.owasp.org/index.php/HTML5_Security_Cheat_Sheet#Local_Storage